distrho/src/DistrhoPluginLV2.cpp

```diff
@@ -1124,6 +1134,18 @@ class PluginLv2
         return ((PluginLv2*)ptr)->writeMidi(midiEvent);
     }
 #endif
+
+#if DISTRHO_PLUGIN_WANT_PARAMETER_VALUE_CHANGE_REQUEST
+    bool setParameterValueChange(const uint32_t index, const float value)
+    {
+		return fCtrlInPortChangeReq->request_change(fCtrlInPortChangeReq->handle, index, value);
```

Comment:
this does not work as intended, because the "true"/sucess value return of this function is 0, not != 0.
so in order to return "true"/success, you need to compare the result to `LV2_CONTROL_INPUT_PORT_CHANGE_SUCCESS`

Also, seems like tabs vs space issue here on this line too